fiat-roles/src/main/java/com/netflix/spinnaker/fiat/providers/BaseResourceProvider.java

```diff
@@ -86,6 +87,12 @@ public void setProviderCacheConfig(ProviderCacheConfig config) {
         .build();
   }
 
+  @Scheduled(fixedRateString = "PT#{${fiat.cache.expires-after-write-seconds:20} / 2}S")
+  public void reloadCache() {
```

Comment:
this just keeps the cache populated - there were occurrences where a user request to login would hit an expired cache and cause the reload to happen in the call flow, causing nasty outliers in response times